apps/passport-server/test/genericIssuance.spec.ts

```diff
@@ -170,6 +177,7 @@ describe("generic issuance service tests", function () {
   let URL_ROOT: string;
   let application: Zupass;
   let giService: GenericIssuanceService | null;
+  let server: SetupServer;
```

Comment:
nit: naming this to something that communicates its a fake pretix backend could be nice